packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -151,7 +151,7 @@ function makeOperationGenerator(optionsParam?: OperationGenerationConfig): Gener
 
     // All subsequent helper functions are generators; note that they don't actually apply any operations.
     function position({ random, sharedString }: ClientOpState): number {
-        return random.integer(0, sharedString.getLength() - 1);
+        return random.integer(0, Math.max(0, sharedString.getLength() - 1));
```

Comment:
@Abe27342 - When the string is empty, this produces an inverted range from 0..-1.  'random-js' is inconsistent in how it handles this.  For integer distributions 'random-js' returns the min (i.e., would always return 0).  For real distributions 'random.js' inverts the range to [-1..0].

I lean towards the 'real' behavior, which falls naturally out of the affine combination '(1 - a) * min + a * max', but since that's a subtle behavioral change, I've made inverted integer ranges an error for the time being.

To preserve the original behavior of clamping to 'min', I added the Math.max(0, ...) above.

However, I was wondering if perhaps 'position' should instead be implemented as:

```ts
return random.integer(0, sharedString.getLength());
```

...which would include the position after the last character of the SharedString?

Reply:
For generating an insert op, `sharedString.getLength()` does seem more correct. But that would break both usages for inclusive/exclusive ranges--ops with index equal to shared-tree length don't pass range validation unless it's an insert (see `getValidOpRange` in `client.ts`).

Probably best to rename this `startPosition` and fix the logic for insert ops to allow end pos === length. I'll do that today if I get some time